dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -282,7 +288,7 @@ private List<Enrollment> getEnrollments( Iterable<Enrollment> enrollments )
             if ( enrollment != null && trackerOwnershipAccessManager
                 .hasAccess( user, enrollment.getTrackedEntity(), enrollment.getProgram() ) )
             {
-                enrollmentList.add( getEnrollment( enrollment, EnrollmentParams.FALSE ) );
+                enrollmentList.add( getEnrollment( enrollment, params ) );
```

Comment:
I think so, I am not sure why this was hardcoded to `EnrollmentParams.FALSE` because when getting enrollments, then the `getEnrollment` method is called and it should return the `fields` requested by the client